src/frontend/components/router-tree/router-tree.ts

```diff
@@ -52,7 +52,7 @@ export class RouterTree {
     };
 
     const nodes = this.tree(d3.hierarchy(
-      root.children.length > 1 ? root : root.children[0], d => d.children));
+      root.children.length === 0 || root.children.length > 1 ? root : root.children[0], d => d.children));
```

Comment:
Just like this, it more for readability for others, not about code correctness :)

```js
root.children.length === 0 || (root.children.length > 1 ? root : root.children[0]), d => d.children));
```

OK so we can have a `boolean` value `(root.children.length === 0)` or an object passed to `d3. hierarchy`, just wanting to make sure as I don't know what this call is doing.